lib/tasks/cleanup.rake

```diff
@@ -18,4 +18,52 @@ namespace :cleanup do
     end
   end
 
+  desc 'Interactively cleanup spam user users'
+  task :spam_users => :environment do
+    spam_scorer = UserSpamScorer.new
+
+    results = {}
+    User.includes(:info_requests).
+      where( "info_requests.user_id IS NULL AND about_me LIKE '%http%' AND ban_text = ''").
+        order("users.created_at DESC").find_each do |user|
+          results[user.id] = spam_scorer.score(user)
+    end
+
+    results.sort_by(&:last).reverse.each do |user_id, spam_score|
+      user = User.find(user_id)
+
+      user.with_lock do
+        display_user(user, spam_score)
+
+        begin
+          puts "Ban this user? (Y/n)"
+          input = $stdin.gets.strip
+        end until %w(Y n).include?(input)
+
+        case input
+        when 'Y'
+          puts "Banning #{ user.id }\n\n"
+          user.update_attributes!(:ban_text => 'Banned for spamming')
+        when 'n'
+          puts "Skipping #{ user.id }\n\n"
+          next
+        end
+      end
+    end
+  end
+
+end
+
+def display_user(user, spam_score)
```

Comment:
I know this could be in a module somewhere, but I'm kinda treating this rake task as temporary until we've actually used it an decided whether we want to work it in to new account creation.